### ISSUE TITLE
Desktop --> Wake doesn't work, AMT --> Wake does

### ISSUE THREAD

**Issue by GenuineJakob** [Ylianst/MeshCentral#4338]
If I try to wake devices that are connected through AMT with the wake button located in the desktop section, nothing happens, the devices stays asleep.
If I go into Intel AMT section and chose "OS Wake from Standby" the devices wakes up.

Video:
[https://youtu.be/CzLJi4iqBSc](https://youtu.be/CzLJi4iqBSc)

**Server Software:**
 - OS: Ubuntu
 - Virtualisation: Docker
 - Network: WAN
 - Version: [e.g. 1.0.60]
 - Node: 16.x
 - Browser: Safari 15.5

**Remote Device:**
 - Device: NUC11TNKv5
 - OS: Windows 11

**Additional context**
An additional question: How am I supposed to add devices with both, AMT and agent? Should I pick the first option, "mange using a software agent" and then add AMT credentials?

**Comment by si458** [comment]
i believe the `wake` option thats located in the desktop tab uses the mesh agent
where as the `wake` option in the AMT uses the AMT
so u cant wake a computer up with the agent because its already asleep, so you have to use the AMT tab instead

**Comment by GenuineJakob** [comment]
Thanks for your reply, from my expierience the mesh agent can't wake devices, at least none of mine.

**Comment by si458** [comment]
> Thanks for your reply, from my expierience the mesh agent can't wake devices, at least none of mine.

that is correct, ive never understand why the `wake-up` option is their because the agent cant wake up the device if its alseep
only the intel AMT computers can physically `wake-up` the devices,
maybe @Ylianst can explain why its their unless its their to indeed `wake-up` the device via AMT and its a bug because its not actually waking it up?
but that wouldnt explain why i see the option when i dont have a single device with AMT support

**Comment by Ylianst** [comment]
So, the "wake-up" action should cause all the other computers on the same device group to send wake-on-lan packets with the MAC addresses of the computer you want to wake up. So, if you have 10 computers on a local network, as long as one is powered on with the agent connected, you should be able to wake the other 9. Obviously, this will only work if wake-on-lan is enabled on these sleeping computers. In addition, if Intel AMT is active, MeshCentral should try both wake-on-lan and Intel AMT wake. So, I will need to check that again, but wake should try all available techniques to wake a remote device.

**Comment by GenuineJakob** [comment]
Thanks for your detailed explanation, to use other agents for wake-up is very smart!
Maybe there is no AMT wake-up because the AMT wake is now „OS wake from standby“ and not just „wake“.

Could you please answer the additional question as well? How am I supposed to add devices with both, AMT and agent? Should I pick the first option, "mange using a software agent" and then add AMT credentials?

**Comment by si458** [comment]
OMG @Ylianst you was right! i managed to wake up a server with WOL even though it was off but my laptop was switched on and has mesh agent running
@GenuineJakob only thing i found was it wouldnt work at first because server 2019 had the default intel drivers installed, BUT remove those drivers then install the drivers directly from intel (my case it was 27.4) and boom the WOL worked a treat!
the server has NO support for AMT at all !
